routes/users.js

```diff
@@ -715,9 +715,9 @@ router.post(
 
 router.get("/:id/listings", async function (req, res, next) {
     const userId = req.params.id;
-    const pageNo = req.query.pageNo || 1;
-    const pageSize = req.query.pageSize || 9;
-
+    const pageNo = Number(req.query.pageNo || 1);
```

Comment:
you can move the or condition outside the number. If anyone sends a string, it will handle that too